packages/plugins/content/image/src/Component/Form/index.js

```diff
@@ -21,74 +21,106 @@
  */
 
 // @flow
-import React from 'react'
+import React, { Component } from 'react'
 import Display from '../Display'
 import FormControlLabel from '@material-ui/core/FormControlLabel'
 import TextField from '@material-ui/core/TextField'
+import Typography from '@material-ui/core/Typography'
 import Checkbox from '@material-ui/core/Checkbox'
 import type { PropTypes } from '../index.js'
+import { ImageUpload } from 'ory-editor-ui'
 
 import { BottomToolbar } from 'ory-editor-ui'
-import { darkTheme } from 'ory-editor-ui/lib/ThemeProvider'
+import { darkTheme } from 'ory-editor-ui/lib/ThemeProvider';
+import type { ImageLoaded, ImageUploaded } from 'ory-editor-ui/lib/ImageUpload';
 
-const handleChange = (onChange: Function) => (e: Event) => {
-  const target = e.target
-  if (target instanceof HTMLInputElement) {
-    const change = {}
+type StateType = {
+  imagePreview?: ImageLoaded
+}
+
+class Form extends Component {
+  state: StateType
+
+  constructor(props: PropTypes) {
+    super(props)
+    this.state = {}
+  }
 
-    if (target.name === 'target') {
-      if (target.checked) {
-        change.target = '_blank'
-        // noopener is safer but not supported in IE, so noreferrer adds some security
-        change.rel = 'noreferrer noopener'
+  handleChange = (onChange: Function) => (e: Event) => {
+    const target = e.target
+    if (target instanceof HTMLInputElement) {
+      const change = {}
+
+      if (target.name === 'target') {
+        if (target.checked) {
+          change.target = '_blank'
```

Comment:
`_blank` is considered [bad practice](https://css-tricks.com/use-target_blank/) and even invalid in some specs (strict XHTML). Can we remove this?

Reply:
This was actually there before. Not sure why it lights up as a change. If you prefer something else, we can change it, but I skimmed through that article and can't say I agree with it 100%. It all seems explained very well, but in some cases, I just want in to open in a new window. I must admit I didn't read the whole thing so maybe that's why.

Reply:
Oh sorry, totally my bad. Let's keep it then, and I'll punch myself in the face for adding it ;) The code shows up as a change because the formatting changed.

Reply:
Haha, no worries, yeah it does, I'll keep an eye on this for the future PRs. It's the vs-code autoformat feature that's doing this. I'll see if I can configure it further

Reply:
Configure it to use `npm run prettier`!